src/xdocs/config_whitespace.xml

```diff
@@ -259,11 +259,19 @@ for (Iterator foo = very.long.line.iterator();
                 <td>6.18</td>
             </tr>
           <tr>
+             <td>treatStaticImportAsImport</td>
+             <td>Allows no empty line between import and static import.</td>
```

Comment:
name of property is completely different with description .... sign of design problem.
Is it only make such effect on validation ?
existing description is hint in what case it might be useful , but not description of property.